src/test/resources/gradle_plugin/pipeline_test_build_scan.txt

```diff
@@ -0,0 +1,12 @@
+stage ('build_Project') {
+    steps {
+        script {
+           if(isUnix()) {
```

Comment:
That is my understanding: When using the tool command then it is automatically added to the path.